packages/core/src/types.ts

```diff
@@ -778,6 +778,8 @@ export type Character = {
             teamAgentIds?: string[];
             teamLeaderId?: string;
             teamMemberInterestKeywords?: string[];
+            disableParseMode?: boolean;
+            parseMode?: "Markdown" | "MarkdownV2" | "HTML";
```

Comment:
Should we use `null | undefined` for disable option, it would be more intuitive
The default value can be set to `null` and people like me can enable it if needed

Reply:
Hmm... I thought it would be appropriate to apply "Markdown" by default and make it able to disable if needed, considering the backward compatibility. But, your suggestion will work assuming that not many users are already using Markdown, or that they can directly recognize the issue and respond to the update within the Developer Community.

Reply:
I've updated!